website/src/app/kb/authenticate/okta/readme.mdx

```diff
@@ -0,0 +1,262 @@
+Okta. [Read more](/kb/authenticate/directory-sync) about how sync works.
+
+## Setup
+
+Setting up the Okta connector is similar to the process of setting up a
+universal OIDC connector. The main difference is the addition of a few extra
+read-only scopes needed to enable sync.
+
+Follow the steps below to setup the Okta connector.
+
+### Step 1: Start the Okta provider setup in Firezone
+
+In your admin portal, go to `Settings` -> `Identity Providers` and click
+`Add Identity Provider`. Then, select `Okta` from the list of identity
+providers.
+
+You'll be shown a summary of the steps you need to complete to setup the Okta
+provider. Keep this page open as you'll need to refer to it in the following
+steps.
+
+### Step 2: Create a new OIDC app in Okta
+
+In your Okta admin portal, go to **Applications** -> **Applications** in the
+left sidebar.
+
+<Link href="/images/kb/authenticate/okta/1-dashboard.png" target="_blank">
+  <Image
+    src="/images/kb/authenticate/okta/1-dashboard.png"
+    alt="Dashboard"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Click **Create App Integration**.
+
+<Link href="/images/kb/authenticate/okta/2-applications.png" target="_blank">
+  <Image
+    src="/images/kb/authenticate/okta/2-applications.png"
+    alt="Applications"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Select **OIDC - OpenID Connect** as the application type.
+
+<Link href="/images/kb/authenticate/okta/3-OIDC-app.png" target="_blank">
+  <Image
+    src="/images/kb/authenticate/okta/3-OIDC-app.png"
+    alt="OIDC app"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Select **Web Application** as the application type and click **Next**.
+
+<Link
+  href="/images/kb/authenticate/okta/4-application-type.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/okta/4-application-type.png"
+    alt="Application type"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Fill in the **App integration name** field with `Firezone Connector`.
+
+[Download the Firezone logo](/images/kb/authenticate/okta/firezone-logo.png) to
+use for the app integration and upload it to Okta (optional).
+
+Ensure the **Refresh Token** grant type is selected.
+
+<Link
+  href="/images/kb/authenticate/okta/5-app-name-grant-type.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/okta/5-app-name-grant-type.png"
+    alt="App name and grant type"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+In the **Sign-in redirect URIs** field, enter the redirect URIs shown in the
+setup form in your Firezone admin portal.
+
+In the **Sign-out redirect URIs** field, enter `https://app.firezone.dev`.
+
+<Link href="/images/kb/authenticate/okta/6-redirect-uris.png" target="_blank">
+  <Image
+    src="/images/kb/authenticate/okta/6-redirect-uris.png"
+    alt="Redirect URIs"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+In the **Assignments** tab, assign the app to the groups you want to have access
+to Firezone.
```

Comment:
Was thinking we probably want to put a note underneath this saying something like:
```
"Please note, the user setting up this connector must have access to the Okta application
being created.  If you choose to "skip group assignment for now", please make sure your 
user is assigned to this Okta app before finishing the setup in Firezone..."
```